gitops-v2/build/main.yaml

```diff
@@ -25,6 +25,7 @@ parameters:
     default: []
   - name: postBuildJob
     type: jobList
+    default: []
```

Comment:
Don't know why is thinks this is new...

Reply:
Have you updated your fork so your new branch matches upstream?